reports/blueprint/report.tex

```diff
@@ -46,10 +46,29 @@
 
     \end{center}
 \end{titlepage}
-\section*{Executive Summary}
-% pop a one pager in here. Ideally less
-% Timothy
-\newpage
+\section{Executive Summary}
+
+This report outlines TeachEE, a device which facilitates online learning for
+electrical engineering labs. It describes the project's scope: a minimum viable
+product for the purpose of this capstone project, with room for scope expansion
+later in the year if circumstances allow it. The minimum viable product consists
+of a oscilloscope, desktop app, and communication between a computer and the
+device via USB. As a general rule, as long as waveforms can be produced in the
+hardware and displayed from software, a minimal product is satisfied.
```

Comment:
Can we replace this general rule sentence with something along the lines of "a complete list of specifications for the MVP are provided in Table X" (the table is somehwere in the doc. use a \ref command in latex.

Reply:
every single reference is broken for me and I can't be bothered to figure it out right now, so please check if the ref correctly renders for you

99b9a9c198547dbcdfbba9e8a20339bfb83165c9